WORKSPACE

```diff
@@ -3292,3 +3293,25 @@ go_repository(
     sum = "h1:t5ivNIH2PK+zw4OBul/iJjsoG9K6kXo4nMDoBpciC8A=",
     version = "v2.3.0",
 )
+
+# Support IVB and later machines.
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+new_git_repository(
```

Comment:
Hi Kevin, we are a little bit confused here about what is expected to do to "maintaining the chunk as a patch". 
Could you please explain a little more on how this external dependency is expected to be maintained and/or probably could we find some similar cases? Thank you so much.

Reply:
There is no similar case. gVisor tries to limit our external dependencies, and those that we utilize tend to be small or from widely used open source projects. This repo, on the other hand, is a substantial chunk of code that seems to be maintained and used exclusively by you.

Keeping this dependency in the repo would:

- Force open source users to pull in a large chunk of code that's not widely used or audited.
- Force the team members at Google to pull this repo into our internal codebase. This can be difficult, especially for a large repo that (again) isn't widely used.

I believe all you'd have to do is maintain this as a ~20 line patch to be applied when syncing with this repo.

Reply:
Hi Kevin, just to make it more clear, as I understand, what we want to do here is to avoid keeping cloning the whole tldk repo into local codebase without checking whether we actually will use plugin stack or not. 
So we probably want to keep the compile process of plugin stack lib in plugin stack repo itself and only export a C library which will be imported as a dependency in cgo. Please correct me if I misunderstand this. Thanks a lot.